doc/passw_hardening/sonic-passwh.yang

```diff
@@ -0,0 +1,91 @@
+module sonic-passwh {
+    yang-version 1.1;
+    namespace "http://github.com/Azure/sonic-passwh";
+	prefix passwh;
+
+    description "PASSWORD HARDENING YANG Module for SONiC OS";
+
+	revision 2021-10-12 {
+        description
+            "First Revision";
+    }
+    typedef feature_state {
+        type enumeration {
+			enum enabled;
+			enum disabled;
+		}
+	}
+
+    container sonic-passwh {
+		container PASSWH {
+			description "PASSWORD HARDENING part of config_db.json";
+			container POLICIES {

```

Comment:
Not sure why do we need nested containers... we can have the table name PASSWD_POLICY for the configurations.